hera_cal/vis_clean.py

```diff
@@ -1412,6 +1412,61 @@ def write_filtered_data(self, res_outfilename=None, CLEAN_outfilename=None, fill
                         self.write_data(data_out, outfilename, filetype=filetype, overwrite=clobber, flags=flags_out,
                                         add_to_history=add_to_history, extra_attrs=extra_attrs, **kwargs)
 
+    def sky_frates(self, to_filter=None, frate_standoff=0.0, frac_frate_sky_max=1.0, min_frate=0.025):
```

Comment:
this would be nice as a standalone function that we can use in sandbox mode. you can still keep a class method sky_frates(self, ...) but let's have that call the standalone function as needed. lmk if that makes sense